pandas/tests/io/parser/common/test_common_basic.py

```diff
@@ -355,7 +355,7 @@ def test_escapechar(all_parsers):
     # https://stackoverflow.com/questions/13824840/feature-request-for-
     # pandas-read-csv
     data = '''SEARCH_TERM,ACTUAL_URL
-"bra tv bord","http://www.ikea.com/se/sv/catalog/categories/departments/living_room/10475/?se%7cps%7cnonbranded%7cvardagsrum%7cgoogle%7ctv_bord"
+"bra tv board","http://www.ikea.com/se/sv/catalog/categories/departments/living_room/10475/?se%7cps%7cnonbranded%7cvardagsrum%7cgoogle%7ctv_bord"
```

Comment:
ikea is swedish and "bra tv bord" translates as "good tv stand". :)

Reply:
Should I ignore this word then?

Reply:
naah, this is OK IMO, let's not ignore more than what's needed, I think this was written more in jest than anything :smile: